app/views/categories/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="navigation">
+  <ul>
+    <li><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+    <h2><%= @category.name %></h2>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+  </ul>
+</div>
+<div class="category-header">
+  <p><%= @category.name %></p>
+  <p>Total Amount: <%= @category.purchases.sum(:amount) %></p>
+</div>
+<div class="purchases-container">
+
+  <% @category.purchases.each do |purchase| %>
+    <div class="purchase-item">
+      <%= link_to purchase.name, category_purchase_path(@category, purchase) %>
+      - Amount: <%= purchase.amount %>
+    </div>
+  <% end %>
+</div>
+<div class="new-category">
+  <div class="new-category-inner">
+    <%= link_to "Add a New Purchase", new_category_purchase_path(@category) %>
+  </div>
+</div>
```

Comment:
- [ ] Based on the requirements, this should be the `Purchases#Index` page because this page should be displayed to users when they click a category. Kindly design the `Purchases#Index` page to look identical to that of the Behance Design provided. Apart from the layout, kindly consider the _**first**_ point in the screenshot shown below:

![image](https://github.com/yegonkimutai/budget-app/assets/77106962/e81e24ab-e94a-43d9-bbc9-9f881f3f2574)

#### N.B: Please note that the _**Add A New Purchase**_  button should be situated at the bottom of the Page as highlighted in the screenshot above. 👍 ⭐ 

| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/misalazeem/Budget-App/assets/77106962/abf231c4-75b0-40dd-b2c8-42ffaf1b73fb) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/47823852-cde4-41a9-b072-1b84ef0109f8) |

#### Hint:
- Redesign the navigation bar
- Order items by the most recent
- Redesign the way each card is displayed
- Position button at the bottom of the page